### PR TITLE
Rehoist! Some reference improvements and fixes.

### PR DESCRIPTION
**Pull request by stristr**
OK to merge.

Short review.

Summary of changes:

- Fixes a user-reported issue where Percy becomes borked on publish—and simplifies the webroot rewrite work by consolidating it with reference rewriting to the migration path. Also, fixes reference rewriting to actually be correct :smile:—because (1) now that `xlink:href` is in the timeline, this code path was never entered for newly instantiated assets and (2) without covering `href`, we technically were not migrating all references in the first place.

Regressions to look for:

- None are expected after careful testing. Reference work is well-covered by QA "problem area" testing, so we should pick up issues during QA if any.

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [x] Updated `changelog/public/latest.json`

### CODE REVIEW

File: changelog/public/latest.json [hunk 1]
```diff
@@ -19,9 +19,10 @@
       "Fixed a bug causing the Actions editor to unexpectedly close.",
       "Fixed a bug preventing Timeline marquee selection to start from within collapsed rows.",
       "Added better logic to calculate the size of the Timeline multiline expressions editor on different sizes.",
-      "Do not allow Timeline rows to be dragged over the controls on the top.",
-      "Focus code editor when switching to code mode",
-      "Fixed wrong result when pressing ctrl during element mouse scaling"
+      "Timeline rows can no longer be dragged over the top controls.",
+      "The code editor is now automatically focused when switching to code mode.",
+      "Pressing Ctrl during element scaling no longer triggers the context menu.",
```
Comment (stristr) [inline changelog/public/latest.json:24]: @jonaias, is this the correct statement of what the problem was? If not, would you please restate what the specific problem was?

**Comment by tsortwehttam** [comment]
Code LGTM.

File: changelog/public/latest.json (hunk 1 above)
Comment (roperzh) [reply changelog/public/latest.json:24]: Thanks for fixing those and sorry for our bad English!

Comment (stristr) [reply changelog/public/latest.json:24]: Your English is not bad! :smile:

(My OCD is very bad though. 😀)

Comment (jonaias) [reply changelog/public/latest.json:24]: @stristr  There was also scale in itself weirdness, but I think it is good enough =)

**Comment by stristr** [comment]
Cool, thanks @matthewtoast and @jonaias. Merging.

**Pull request merged**